### PR TITLE
fix resolutions issues

### PR DESCRIPTION
**Pull request by JoeNunnelley**
several resolution issues needing to be resolved.

### CODE REVIEW

File: project/build.sbt [hunk 1]
```diff
@@ -1,5 +1,12 @@
 resolvers ++= Seq(
-  "socrata releases" at "https://repo.socrata.com.com/libs-release"
+  "socrata releases" at "https://repo.socrata.com/libs-release",
+  Resolver.url("typesafe sbt-plugins", url("https://dl.bintray.com/typesafe/sbt-plugins"))(Resolver.ivyStylePatterns)
+)
+
+libraryDependencies ++= Seq(
+  "org.scalacheck" %% "scalacheck" % "1.11.3" % Test,
+  "org.scala-lang" % "scala-library" % "2.10.6",
+  "org.apache.httpcomponents" % "httpclient" % "4.3.3"
```
Comment (rjmac) [inline project/build.sbt]: I don't think these should be necessary?  It would be very weird to me if they were!

Comment (JoeNunnelley) [reply project/build.sbt]: without those I was getting resolution issues.  ```[warn]   http://repo.typesafe.com/typesafe/ivy-releases/org.scala-lang/scala-library/2.10.6/jars/scala-library.jar
[warn] ==== sbt-plugin-releases: tried
[warn]   http://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/org.scala-lang/scala-library/2.10.6/jars/scala-library.jar
[warn] ==== local: tried
[warn]   /Users/joe.nunnelley/.ivy2/local/org.scala-lang/scala-library/2.10.6/jars/scala-library.jar
[warn] ==== local-preloaded-ivy: tried
[warn]   file:/Users/joe.nunnelley/.sbt/preloaded/org.scala-lang/scala-library/2.10.6/jars/scala-library.jar
[warn] ==== local-preloaded: tried
[warn]   file:////Users/joe.nunnelley/.sbt/preloaded/org/scala-lang/scala-library/2.10.6/scala-library-2.10.6.jar
[warn] ==== public: tried
[warn]   http://repo1.maven.org/maven2/org/scala-lang/scala-library/2.10.6/scala-library-2.10.6.jar
[warn] ==== socrata releases: tried
[warn]   https://repo.socrata.com/libs-release/org/scala-lang/scala-library/2.10.6/scala-library-2.10.6.jar
[warn] ==== typesafe sbt-plugins: tried
[warn]   https://dl.bintray.com/typesafe/sbt-plugins/org.scala-lang/scala-library/2.10.6/jars/scala-library.jar
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::              FAILED DOWNLOADS            ::
[warn] 	:: ^ see resolution messages for details  ^ ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: org.scala-lang#scala-library;2.10.6!scala-library.jar
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
sbt.ResolveException: download failed: org.scala-lang#scala-library;2.10.6!scala-library.jar
	at sbt.IvyActions$.sbt$IvyActions$$resolve(IvyActions.scala:217)
	at sbt.IvyActions$$anonfun$update$1.apply(IvyActions.scala:126)
	at sbt.IvyActions$$anonfun$update$1.apply(IvyActions.scala:125)
	at sbt.IvySbt$Module$$anonfun$withModule$1.apply(Ivy.scala:115)
	at sbt.IvySbt$Module$$anonfun$withModule$1.apply(Ivy.scala:115)
	at sbt.IvySbt$$anonfun$withIvy$1.apply(Ivy.scala:103)```

Comment (rjmac) [reply project/build.sbt]: Ah, I see.  I think there's a better fix than this, though.  Just a moment (or rather several moments because :snail:)

Comment (rjmac) [reply project/build.sbt]: (the root problem is that this SBT is old enough to want to talk to Maven Central over http instead of https)

Comment (JoeNunnelley) [reply project/build.sbt]: pulled down your change and was able to build locally.

**Pull request merged**